src/main/scala/nl.knaw.dans.easy.bag2deposit/ddm/DropEmptyRewriteRule.scala

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bag2deposit.ddm
+
+import scala.xml.Node
+import scala.xml.transform.RewriteRule
+
+object DropEmptyRewriteRule extends RewriteRule {
+  private val labels = Seq(
+    "linkedRelation",
+    "conformsTo",
+    "hasFormat",
+    "hasPart",
+    "hasVersion",
+    "isFormatOf",
+    "isPartOf",
+    "isReferencedBy",
+    "isReplacedBy",
+    "isRequiredBy",
+    "isVersionOf",
+    "references",
+    "replaces",
+    "requires",
+  )
+
+  override def transform(node: Node): Seq[Node] = {
+    if (node.text.trim.isEmpty && labels.contains(node.label))
```

Comment:
Shouldn't I log empty relations that have an href and leave them as is? There might be a fix for these mistakes.

Reply:
an empty relation is not valid. We could opt for repeating the href value in the text?
But my guess is that that is not usually the case.

Reply:
Saw something like that on jump-off pages, Might be unlikely in this context but already done.